tests/logging/counterTestPod.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: counter-test-pod
+spec:
+  containers:
+  - name: count
+    image: ubuntu:14.04
+    args: [bash, -c, 
+           'for ((i = 0; ; i++)); do echo "$i: oklogTest-$(date)"; sleep 1; done']
```

Comment:
As this pod will push logs as long as it is alive, why not make something like
```'for ((i = 0; i<10; i++)); do echo "$i: oklogTest-$(date)"; sleep 2; done']```
This serves well for testing.

Reply:
done